.vib/mediawiki/cypress/cypress/integration/mediawiki_spec.js

```diff
@@ -45,29 +22,15 @@ it('allows to create a new page', () => {
 it('allows to upload a file', () => {
   cy.login();
   cy.visit('/wiki/Special:Upload');
-  cy.get('#wpUploadFile').selectFile('cypress/fixtures/images/post_image.png', {
+  cy.get('#wpUploadFile').selectFile('cypress/fixtures/images/test_image.jpeg', {
     force: true,
   });
   // We'll use a random fileName to bypass duplication-related errors
-  cy.get('input[name="wpDestFile"]').type(`${random}.png`);
+  cy.get('[name="wpDestFile"]').clear().type(`testfile-${random}.jpeg`);
   // If Mediawiki detects that an identical file was already uploaded
-  // it will ask for confirmation, requiring additional steps
-  cy.get('input[name="wpIgnoreWarning"]').click();
-  cy.get('input[name="wpUpload"]').click();
-  cy.contains('h1', `File:${random}.png`, { matchCase: false });
+  cy.get('[name="wpIgnoreWarning"]').click();
+-  cy.get('[name="wpUpload"]').click();
+-  cy.contains(`testfile-${random}.jpeg`, { matchCase: false }); // it will ask for confirmation, requiring additional steps
```

Comment:
I think this line is duplicated code with the next one (`34`). You are basically checking for the existence of "testfile-${random}.jpeg", but using a less restrictive selector than in 34.